src/core/router/CustomerPortalRoutes.tsx

```diff
@@ -8,10 +8,20 @@ const PortalInit = lazyLoad(
 
 // ----------- Routes -----------
 export const CUSTOMER_PORTAL_ROUTE = '/customer-portal/:token'
+export const CUSTOMER_PORTAL_ROUTE_PAGE = '/customer-portal/:token/:page'
```

Comment:
Why do we prefer having a list of page and trigger their render in the DOM with `if page === 'xxx' => render xxx` rather than having a route definition for each pages?

Reply:
Done, I have removed the extra routes, included an Outlet and did it through react-router

Reply:
Yeah I think it's way better this way! 🙌